src/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -889,8 +889,12 @@ export class BaseExtension implements IExtension {
         return null;
     }   
 
-    getIIIFShareUrl(): string {
-        return this.helper.iiifResourceUri + "?manifest=" + this.helper.iiifResourceUri;
+    getIIIFShareUrl(shareManifests: boolean = false): string {
+        if (shareManifests) {
+          return `${this.helper.manifest.id}?manifest=${this.helper.manifest.id}`;
```

Comment:
What is the value of `this.helper.manifest.id` in the case that you're sharing a manifest (and `shareManifests` is false) Is it the same as `this.helper.iiifReourceUri`. Should (or can) we always use `this.helper.manifest.id`?

Reply:
`this.helper.manifest.id` consistently links to a `sc:Manifest` resource, while `this.helper.iiifResourceUri` can return either a link to `sc:Manifest` or a `sc:Collection`. 
 For example:

http://universalviewer.io/examples/#?c=&m=&s=&cv=35&manifest=https%3A%2F%2Ffiggy.princeton.edu%2Fconcern%2Fscanned_resources%2Fabcbea26-6600-4445-be9f-bf0ac0ba965a%2Fmanifest&xywh=0%2C-638%2C7199%2C6500
generates a share URL linking to
https://figgy.princeton.edu/concern/scanned_resources/abcbea26-6600-4445-be9f-bf0ac0ba965a/manifest?manifest=https://figgy.princeton.edu/concern/scanned_resources/abcbea26-6600-4445-be9f-bf0ac0ba965a/manifest)

When a `sc:Collection` is drag-and-dropped into Mirador (https://projectmirador.org/demo/), it breaks.